### DISCUSSION TITLE
[1726130]无径之林 {0713最新本体}

### DISCUSSION THREAD

**Discussion by w414382739** [pjy612/SteamManifestCache#756] [Share]
### Steam 游戏ID

1726130

### 密钥列表

```hash
1726131:b2a6e1b03d3993d322e9df6ce92c370c2eded9c6311e4af8e6ed287473606f11
```

### 清单附件

ManifestID: 8324406637317545536 Size: 3.01 GB
[1726130.zip](https://github.com/user-attachments/files/16484609/1726130.zip)

### 备注

XCOM2不知道为什么steam打不开了，分享一个无径之林，之前耽误大佬时间了，这次是用永久获取工具生成的，应该没问题

email:w414382739@163.com

**Comment by w414382739** [comment]
@pjy612 大佬，想用SAI下游戏，麻烦大佬告知一下秘钥怎么获取才能用下载每天更新清单的内容，需要赞助，我就赞助，需要分享我就分享

**Reply by w414382739** [reply]
买游戏太贵了，我还是学生，真的买不起啊

**Reply by pjy612** [reply]
...学生不应该先好好学习么...
更新记录只是拿着看的
tools 论坛里面有别人分享的 能玩就行

**Reply by w414382739** [reply]
@pjy612 大学，时间比较多，大佬赞助可以吗，其实想玩绝命游歌和废土快递，论坛上没有，我搜了

**Reply by pjy612** [reply]
那你可以论坛悬赏啊。。。

**Reply by w414382739** [reply]
@pjy612 谢谢大佬，那我自己在冲浪找一找吧，我想赞助，弄个内测也不行吗？祝大佬身体健康

**Reply by pjy612** [reply]
这是个免费项目，为什么目前如此，置顶有写。
赞助也不是给我，我搞这项目也不是为了赚钱。
这个版块留着就有它的意义，至于其他的，随缘，缘分到了自然就

**Comment by w414382739** [comment]
@pjy612 大佬球球了，看一眼，感谢感谢
